test/iron-dropdown.html

```diff
@@ -316,14 +333,14 @@
                   getComputedStyle(content).pointerEvents,
                   'none',
                   'content is interactive');
+              dropdown.addEventListener('iron-overlay-opened', function() {
```

Comment:
I moved this in because this test passes locally when the animation fails to load otherwise. Particularly, the 'iron-overlay-opened' event is fired immediately and the assertion here passes since it didn't change. Later, outside of the test, the timeout fires and throws, but this doesn't fail the test.